### PR TITLE
[valgrind] Use libtool to not catch bash related issues

### PR DESCRIPTION
**Pull request by ebraminio**

### CODE REVIEW

**Comment by behdad** [comment]
Don't run `which libtool`. Use the one from our topbuilddir.

**Comment by behdad** [comment]
Would also help not finding OS X's unrelated libtool.

File: test/fuzzing/run-shape-fuzzer-tests.py [hunk 1]
```diff
@@ -68,11 +68,17 @@ def timeout(p, is_killed):
...
 valgrind = None
+libtool = None
 if os.environ.get('RUN_VALGRIND', ''):
 	valgrind = which ('valgrind')
 	if valgrind is None:
 		print ("""Valgrind requested but not found.""")
 		sys.exit (1)
+	LIBTOOL = os.environ.get('LIBTOOL')
+	if LIBTOOL:
+		libtool = LIBTOOL.split(' ')[-1]
```
Comment (ebraminio) [inline test/fuzzing/run-shape-fuzzer-tests.py]: Hack, as it was returning something '/bin/sh ../libtool' and I wanted the later part.

File: test/fuzzing/run-subset-fuzzer-tests.py [hunk 2]
```diff
@@ -20,13 +39,32 @@
...
+		print ("""Valgrind requested but not found.""")
+		sys.exit (1)
+	LIBTOOL = os.environ.get('LIBTOOL')
+	if LIBTOOL:
+		libtool = LIBTOOL.split(' ')[-1]
+	else:
+		libtool = which ('libtool')
+
+# Disable it for now
+valgrind = None
```
Comment (ebraminio) [inline test/fuzzing/run-subset-fuzzer-tests.py]: Disabled as it fails with this
image

File: test/fuzzing/run-shape-fuzzer-tests.py (hunk 1 above)
Comment (behdad) [reply test/fuzzing/run-shape-fuzzer-tests.py]: Why?  Just leave it as is.

Comment (behdad) [reply test/fuzzing/run-shape-fuzzer-tests.py]: I mean, split it and prepend that to the args list...

**Comment by ebraminio** [comment]
Let's merge this for now I think

**Comment by behdad** [comment]
Yeah, whatever.  If it works, sure.

**Comment by behdad** [comment]
Open issue re removing python test runner?

**Comment by behdad** [comment]
I guess if we want to catch timeouts, we would need some kind of test runner.

**Pull request merged**